src/EditorFeatures/VisualBasicTest/RemoveUnusedVariable/RemoveUnusedVariableTest.vb

```diff
@@ -73,6 +73,225 @@ Module M
+            Return L()
+        End Function
+    End Class
+End Module
+</File>
+
+            Await TestAsync(markup, expected)
+        End Function
+
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnusedVariable)>
+        Public Async Function JointDeclarationRemoveFirst() As Task
+            Dim markup =
+<File>
+Module M
+    Function F() As Integer
+        Dim [|a|] As Integer, b As Integer
+        Return b
+    End Function
+End Module
+</File>
+            Dim expected =
+<File>
+Module M
+    Function F() As Integer
+        Dim b As Integer
+        Return b
+    End Function
+End Module
+</File>
+
+            Await TestAsync(markup, expected)
+        End Function
+
+        <Fact(Skip:="https://github.com/dotnet/roslyn/issues/24076"), Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnusedVariable)>
+        Public Async Function JointDeclarationAndAssignmentRemoveFirst() As Task
+            Dim markup =
+<File>
+Module M
+    Function F() As Integer
+        Dim [|a|] As Integer = 0, b As Integer = 0
+        Return b
+    End Function
+End Module
+</File>
+            Dim expected =
+<File>
+Module M
+    Function F() As Integer
+        Dim b As Integer
+        Return b
+    End Function
+End Module
+</File>
+
+            Await TestAsync(markup, expected)
+        End Function
+
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnusedVariable)>
+        Public Async Function JointDeclarationRemoveSecond() As Task
+            Dim markup =
+<File>
+Module M
+    Function F() As Integer
+        Dim a As Integer, [|b|] As Integer
+        Return a
+    End Function
+End Module
+</File>
+            Dim expected =
+<File>
+Module M
+    Function F() As Integer
+        Dim a As Integer
+        Return a
+    End Function
+End Module
+</File>
+
+            Await TestAsync(markup, expected)
+        End Function
+
+        <Fact(Skip:="https://github.com/dotnet/roslyn/issues/24076"), Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnusedVariable)>
+        Public Async Function JointDeclarationAndAssignmentRemoveSecond() As Task
+            Dim markup =
+<File>
+Module M
+    Function F() As Integer
+        Dim a As Integer = 0, [|b|] As Integer = 0
+        Return a
+    End Function
+End Module
+</File>
+            Dim expected =
+<File>
+Module M
+    Function F() As Integer
+        Dim a As Integer
+        Return a
+    End Function
+End Module
+</File>
+
+            Await TestAsync(markup, expected)
+        End Function
+
+        <Fact(Skip:="https://github.com/dotnet/roslyn/issues/24076"), Trait(Traits.Feature, Traits.Features.CodeActionsRemoveUnusedVariable), Trait(Traits.Feature, Traits.Features.CodeActionsFixAllOccurrences)>
+        Public Async Function JointDeclarationAndAssignmentRemoveBoth() As Task
+            Dim markup =
+<File>
+Module M
+    Sub F()
+        Dim {|FixAllInDocument:a as Integer|} = 0, b As Integer = 0
```

Comment:
Can we have at least one unskipped FixAll test for VB?

Reply:
There were already provided before this PR. PLease see above for  Line 59:
Public Async Function RemoveUnusedVariableFixAll() As Task



---
In reply to: [161330426](https://github.com/dotnet/roslyn/pull/24077#discussion_r161330426) [](ancestors = 161330426)